sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -67,6 +76,155 @@ describe(
       );
     });
 
+    it("Should create a notification for the ministry and student for a blocked disbursement when there are no previously existing notifications for the disbursement.", async () => {
+      // Arrange
+      const { student, disbursementId } = await createTestData();
+      // Queued job.
+      const mockedJob = mockBullJob<void>();
+
+      // Act
+      const result = await processor.processECert(mockedJob.job);
+
+      // Assert
+      expect(result).toStrictEqual([
+        "Process finalized with success.",
+        "Generated file: none",
+        "Uploaded records: 0",
+      ]);
+      expect(
+        mockedJob.containLogMessages([
+          `Creating notifications for disbursement id: ${disbursementId} for student and ministry.`,
+          `Completed creating notifications for disbursement id: ${disbursementId} for student and ministry.`,
+        ]),
+      ).toBe(true);
+      const notifications = await db.notification.find({
+        select: {
+          id: true,
+          user: { id: true },
+          notificationMessage: { id: true },
+        },
+        relations: { user: true, notificationMessage: true },
+        where: {
+          metadata: {
+            disbursementId,
+          },
+          dateSent: IsNull(),
+        },
+        order: { notificationMessage: { id: "ASC" } },
+      });
+      notifications.map((notification) => delete notification.id);
```

Comment:
I would not encourage deleting the id property. What about not adding id to select ?? or applying expect on individual properties??